types/src/time.rs

```diff
@@ -0,0 +1,17 @@
+use instant::SystemTime;
+use tendermint::Time;
+
+// rely on instant to provide time for wasm32
+pub(crate) fn now() -> Time {
```

Comment:
Usage of time in tendermint was causing panics so I've added this compatibility layer. Is it the right place? Or maybe this should be moved to our tendermint fork entirely?

Reply:
I think since we do not wrap `Time` and replace it in a new `Header`. The best option is to fix it in tendermint fork. Users may want to construct their own `ExtendedHeader` and they will use it.

Reply:
Moved the fix to tendermint: https://github.com/eigerco/celestia-tendermint-rs/pull/13